vmm/sandbox/src/main.rs

```diff
@@ -158,6 +158,10 @@ async fn main() -> anyhow::Result<()> {
     }
     builder.init();
 
+    // notify the systemd about state changes
+    #[cfg(feature = "systemd")]
+    sd_notify::notify(false, &[sd_notify::NotifyState::Ready])?;
```

Comment:
Should we ignore the error returned by `notify` and keep on running? Probably just record this error in logs.
So that we don't need the "systemd" feature.

Reply:
I've thought about this, but if the `sd_notify` execution fails to continue down the line, then `systemctl start kuasar-vmm` systemd service will  be blocked forever and `kuasar-vmm` systemd service will abnormal.

Reply:
I found both [dockerd ](https://github.com/moby/moby/blob/master/cmd/dockerd/daemon_linux.go#L18) and [containerd ](https://github.com/containerd/containerd/pull/4088/files) all ignored this error.

Reply:
> I found both [dockerd ](https://github.com/moby/moby/blob/master/cmd/dockerd/daemon_linux.go#L18) and [containerd ](https://github.com/containerd/containerd/pull/4088/files) all ignored this error.

If we just ignore the error with `notify` type systemd service, which will cause `systemctl start` command blocked until exceeding the timeout.